### DISCUSSION TITLE
unable to locate *** in dev

### DISCUSSION THREAD

**Discussion by QMangoJ** [flareact/flareact#252] [Q&A]
run dev failed
how can i do to run dev? please

flareact -V 1.2.1
wrangler -V 1.20.0

Starting Flareact dev server on http://localhost:8080 ...

[worker] ⚠ ｢copy-webpack-plugin｣: unable to locate '/Users/bytedance/Desktop/projects/my-project-flareact/public' at '/Users/bytedance/Desktop/projects/my-project-flareact/public'
[worker] Error: HTTP status client error (403 Forbidden) for url (https://api.cloudflare.com/client/v4/accounts/777d1360a663b0b3e272729282e123ff/workers/subdomain/edge-preview)
[worker] wrangler dev exited with code 1
--> Sending SIGTERM to other processes..
[client] webpack-dev-server --config node_modules/flareact/configs/webpack.client.config.js --mode development --port 8080 exited with code 0
[
  {
    command: {
      command: 'wrangler dev',
      name: 'worker',
      prefixColor: '',
      env: [Object]
    },
    index: 0,
    exitCode: 1
  },
  {
    command: {
      command: 'webpack-dev-server --config node_modules/flareact/configs/webpack.client.config.js --mode development --port 8080',
      name: 'client',
      prefixColor: '',
      env: [Object]
    },
    index: 1,
    exitCode: 0
  }
]

**Comment by tj-kev** [comment]
Could you try manually creating the public folder to see if that solves your problem?
